### PR TITLE
Display claim information

### PR DESCRIPTION
**Pull request by TaiWilkin**
## Overview

Ports the facility claim information section from the old sidebar to
the new sidebar, removes old sidebar components, and adds a 
claimed badge to claim items. 

Connects #1765 

## Demo

Screen Shot 2022-06-14 at 9 18 57 AM
Screen Shot 2022-06-14 at 9 19 08 AM
Screen Shot 2022-06-14 at 9 19 29 AM
Screen Shot 2022-06-14 at 9 19 52 AM

## Testing Instructions

* Run `./scripts/server`
* Visit an unclaimed facility and confirm that the details load as before. 
* Claim the facility. Approve the claim as an admin user. 
* Visit the claimed facility and view the additional 'description' field. Scroll over the badge to see the tooltip.
* Fill out the full claim form and make all details public. View the claimed facility and confirm the name, address, and other claim fields are integrated and show badges. 
* Make some details private in the claim form. View the claimed facility and confirm those fields are hidden.
* The claim feature is disabled in embed mode. View the claimed facility in an embedded map. Confirm that the claim badges and claim fields are hidden.

## Checklist

- [x] `fixup!` commits have been squashed
- [x] CI passes after rebase
- [x] CHANGELOG.md updated with summary of features or fixes, following [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) guidelines
- [x] This PR is targeted at the correct branch (`develop` vs. `ogr/develop`)
- [x] If this PR applies to both OAR and OGR a companion PR has been created

### CODE REVIEW

File: src/app/src/components/FacilityDetailSidebarClaimedInfo.jsx [hunk 1]
```diff
@@ -235,10 +209,10 @@ FacilityDetailsSidebarClaimedInfo.propTypes = {
         id: number.isRequired,
         facility: shape({
             description: string.isRequired,
-            name: string.isRequired,
+            name_english: string.isRequired,
             address: string.isRequired,
-            website: string.isRequired,
```
Comment (TaiWilkin) [inline src/app/src/components/FacilityDetailSidebarClaimedInfo.jsx:240]: Some of the props are null if the user has chosen to keep them private.

File: src/app/src/components/BadgeVerified.jsx [hunk 2]
```diff
@@ -2,9 +2,9 @@ import React from 'react';
 import { string, shape } from 'prop-types';
 import SvgIcon from '@material-ui/core/SvgIcon';
 
-export default function BadgeVerified({ color, style }) {
+export default function BadgeVerified({ color, style, ...props }) {
```
Comment (TaiWilkin) [inline src/app/src/components/BadgeVerified.jsx:5]: The props for the MUI tooltip need to be passed down to the inner components

**Review by jwalgran — APPROVED** [review]
Great, clear implementation based on a good design suggestion. Thanks for working through that design change with the OAR team.

In addition to the written testing instructions I verified this by filling in every field on an approved claim for "1v1 Fight Gear", and also POSTing  this as different contributor and vierified that bothe "badged" and "unbadged" extended fields appeared in the sidebar 

```
{
"name":"1v1 Fight Gear",
"address": "5320 Tower Way, Sanford, 32773, Florida ",
"country":"us",
"number_of_workers": 2,
"facility_type_processing_type": "sew",
"parent_company":"api parent co",
"product_type": ["api a", "api b"]
}
```

I encountered bug #1903 while running this test but it is unrelated to the changes in this PR.

**Comment by TaiWilkin** [comment]
Thank you for reviewing!

**Pull request merged**